lib/agoneum/web/templates/registration/new.html.slim

```diff
@@ -0,0 +1,29 @@
+h1 Create an Account
+
+= form_for @changeset, registration_path(@conn, :create), fn f ->
+  = if f.errors != [] do
+    div.alert.alert-danger
+      p Oops, something went wrong! Please check the errors below:
+
+  div.form-group
+    = label(f, :email, class: "control-label")
+    = text_input(f, :email, class: "form-control")
+    = error_tag(f, :email)
+
+  div.form-group
+    = label(f, :name, "Display Name", class: "control-label")
+    = text_input(f, :name, class: "form-control")
+    = error_tag(f, :name)
+
+  div.form-group
+    = label(f, :password, class: "control-label")
+    = password_input(f, :password, class: "form-control")
+    = error_tag(f, :password)
+
+  div.form-group
+    = label(f, :password_confirmation, "Password Confirmation", class: "control-label")
+    = password_input(f, :password_confirmation, class: "form-control")
+    = error_tag(f, :password_confirmation)
+
+  div.form-group
+    = submit("Sign Up", class: "btn btn-primary")
```

Comment:
We don't have to do it on this PR, but a practice I've grown to like is, when making view changes, to attach a pic of what things look like on the PR as well :)